bot.py

```diff
@@ -25,26 +34,39 @@ async def format_embed(self, name, anime=False):
         except kadal.MediaNotFound:
             return
 
-        title = (media.title.get('english')
-                 or media.title.get('romaji')
-                 or media.title.get('native'))
-        desc = "***" + ", ".join(media.genres) + "***\n"
+        title = (media.title.get("english")
+                 or media.title.get("romaji")
+                 or media.title.get("native"))
+        desc = "**" + ", ".join(media.genres) + "**\n"
         if media.description is not None:
             desc += media.description[:256 - len(desc)] + f"... [(more)]({media.site_url})"
         # dirty half-fix until i figure something better out
-        desc = desc.replace("<br>", "").replace("<i>", "").replace("</i>", "")
-        footer = re.sub(r'.*\.', '', str(media.format))
-        e = discord.Embed(title=title, description=desc, color=0x4286f4)
-        e.set_footer(text=footer.replace("TV", "ANIME"))
-        e.set_thumbnail(url=media.cover_image)
+        desc = re.sub(r"<[bri/]{1,2}>", "", desc, flags=re.I|re.M)
+        footer = re.sub(r".*\.", "", str(media.format))
+        
+        async with aiohttp.ClientSession() as session:
+            async with session.get(f"{self.anilist_cover_url}{media.id}") as resp:
+                if resp.status == 200:
+                    f = await aiofiles.open("image.tmp", mode='wb')
+                    await f.write(await resp.read())
+                    await f.close()
+                    palette_color = await asyncio.get_running_loop().run_in_executor(None, functools.partial(fast_colorthief.get_palette, "image.tmp", color_count=2, quality=100))
```

Comment:
Line is getting long, consider splitting it out to variables, ie loop, partial func, then assembling to run_in_executor.
Also, since you use it multiple times put the image name in a variable

Reply:
:notlikethis:
How about we just beautify the line a little
```py
palette_color = await asyncio.get_running_loop().run_in_executor(
    None,
    functools.partial(
        fast_colorthief.get_palette,
        temp_img,
        color_count=2,
        quality=100
    )
)
```

Reply:
that works in the line limit but i really dislike it
also you can get the loop from the bot itself like so
```python
palette_partial = functools.partial(fast_colorthief.get_palette, "image.tmp", color_count=2, quality=100)
self.loop.run_in_executor(None, palette_partial)
```
can still break down the partial but thats up to you